index.js

```diff
@@ -0,0 +1,19 @@
+const {
+  exec
+} = require("child_process");
+const {
+  parseAuditData
+} = require('./parser')
+
+exec("npm audit --json > ./scans/npmaudit.json", (error, stdout, stderr) => {
```

Comment:
So, a thing to note here, is that we need to figure out how to make a bin file so that auditly can be run from the command line in any project.  This would currently run npm audit against the auditly project itself.   We need to be able to run it against any other project.

Reply:
Will look into it